.github/workflows/Linux.yml

```diff
@@ -101,35 +102,40 @@ jobs:
           # Check if there are lua classes with invalid/improper declarations.
           python3 scripts/check_lua_classes.py
           # Check for tabs (should be spaces)
-          grep -Ire '\t' CorsixTH/Lua --exclude-dir=languages
+          ! grep -IrnP '\t' CorsixTH/Lua
           # Run CPP Tests
           cd build/CorsixTH
           ctest --extra-verbose --build-config Release --output-on-failure
           cd ../..
           # Build and package LevelEdit
           ant -buildfile LevelEdit/build.xml dist
-          # Run codespell
-          codespell --enable-colors --quiet-level 2 --skip="languages,corsix-th.6" \
-            -L sav,unexpect,persistance,defin,uint,inout,currenty,blong \
-            AnimView CorsixTH CorsixTH/Lua/languages/english.lua LevelEdit libs
+          # Run codespell (temporarily not a failure, Nov 2023)
```

Comment:
Why not enable it ?

Reply:
Temporary non-failure state to bypass some current PRs according to the pr summary. It'll be put back again later once the current stack of PRs is merged.